build/webpack.common.config.js

```diff
@@ -63,6 +63,7 @@ module.exports = language => {
         ],
         resolve: {
             alias: {
+                'box-annotations-messages': path.resolve(`node_modules/box-annotations/i18n/${language}`),
```

Comment:
Are we able to use the TranslationPlugin's `additionalMessageData` option for this?

Reply:
Not at this time, we have to move our current i18n to the root level. The plugin expects the project structure to follow
projectRoot/
 *  i18n/
 * *  json/
 * * bundles/
 **   *.properties
 *  src/